App.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { store } from "./modules/store/index";
```

Comment:
Not needed to use /index in import

Reply:
This current configuration isn't supporting by expo